RecoJets/JetProducers/python/PileupJetID_cfi.py

```diff
@@ -10,8 +10,9 @@
 _chsalgos_74x = cms.VPSet(full_74x_chs,cutbased)
 _chsalgos_76x = cms.VPSet(full_76x_chs,cutbased)
 _chsalgos_80x = cms.VPSet(full_80x_chs,cutbased)
+_chsalgos_81x = cms.VPSet(full_81x_chs,cutbased)
 
-_stdalgos    = _chsalgos_80x
+_stdalgos    = _chsalgos_81x
```

Comment:
the "81X" training is based on 80X RECO. The naming "81X" may indeed confuse. It is rather "80XvarFix", like the name of the weight files. However, this name is never visible anywhere to the user.
Is it worth changing the name, but breaking consistency of 80X and 81X releases?